pytest_plt/tests/test_pytest.py

```diff
@@ -0,0 +1,61 @@
+"""The main test file that should be run on a regular basis.
+
+The goal of this file is to test pytest_plt internals and run all other
+test files with various invocations of pytest to ensure that all possible
+ways of using this plugin are tested.
+
+By default, if you call ``pytest`` while in this repository, only this
+file will be run due to configuration in ``setup.cfg``. However, other
+test files can be run manually by passing them to ``pytest``.
+"""
+
+import os
+
+from pytest_plt import Mock
+
+pytest_plugins = ["pytester"]
+
+
+def test_mock():
+    mock = Mock()
+    assert isinstance(mock["item"], Mock)
+    assert mock * "Hi" == 1.0
+    assert mock.__file__ == "/dev/null"
+    assert mock.__file__ == mock.__path__
+    assert mock.Type.__module__ == "pytest_plt"
+    assert type(mock.Type) is type
+
+
+def test_plt_no_plots(testdir):
+    testdir.mkpydir("package")
+    testdir.mkpydir("package/tests")
+    file_path = testdir.copy_example("test_plt.py")
+    file_path.rename("package/tests/test_plt.py")
+
+    result = testdir.runpytest()
+    result.assert_outcomes(passed=3)
```

Comment:
This is 100% of the tests, right? I'm wondering if we can check that, somehow, rather than relying on the absolute number. I'll look into it a bit.